README.md

```diff
@@ -11,6 +11,8 @@ relevant branch e.g. `5` will be used depending on the command-line `--branch` o
 It will run all scripts in the `scripts/any` folder and then run all scripts in the applicable 
 `scripts/<cms-version>` folder depending on the command-line `--branch` option that's passed in.
 
+If the `--unsupported` option is used then it will only run scripts in the `scripts/cms-unsupported` folder.
+
```

Comment:
```suggestion
```
This is mentioned further down (with the correct name)

Reply:
Done